tests/unit/email/test_services.py

```diff
@@ -101,10 +101,18 @@ def test_creates_service(self):
         assert sender._sender == "DevPyPI <noreply@example.com>"
         assert sender._db is request.db
 
-    def test_send(self, db_session):
+    def test_send(self, db_session, monkeypatch):
+        # Determine what the random boundary token will be
+        import random
+        import sys
+
+        random.seed(42)
+        token = random.randrange(sys.maxsize)
+        random.seed(42)
```

Comment:
should we rescramble the seed after the test? `random` is bad, but I assume other things are using it in the codebase and we probably want to either prefer deterministic pseudorandom behavior... or pure pseudorandom behavior for our suite.

Reply:
not a blocker, just a curio.

Reply:
I don't think it's necessary!